assets/global.js

```diff
@@ -456,15 +456,17 @@ class SliderComponent extends HTMLElement {
   }
 
   initPages() {
-    if (!this.sliderItems.length === 0) return;
-    this.slidesPerPage = Math.floor(this.slider.clientWidth / this.sliderItems[0].clientWidth);
-    this.totalPages = this.sliderItems.length - this.slidesPerPage + 1;
+    this.sliderItemsToShow = Array.from(this.sliderItems).filter(element => element.clientWidth > 0);
```

Comment:
I think this should be a `const`. This value is not used by any other function in this class.

```suggestion
    const sliderItemsToShow = Array.from(this.sliderItems).filter(element => element.clientWidth > 0);
```

Reply:
👌  done